### ISSUE TITLE
[Question] Multiple instances- data integrity

### ISSUE THREAD

**Issue by SRGOM** [abraunegg/onedrive#775]
Should I worry about data integrity if I accidentally had multiple instances (2) of onedrive services running on my computer?

Little worried that I'm seeing messages like `Processing 215 OneDrive items to ensure consistent state due to sync_list being used` in one of the instances after I launched it (I launched it before killing another one)

**Comment by abraunegg** [comment]
@SRGOM 


> Should I worry about data integrity if I accidentally had multiple instances (2) of onedrive services running on my computer?
> 
> Little worried that I'm seeing messages like `Processing 215 OneDrive items to ensure consistent state due to sync_list being used` in one of the instances after I launched it (I launched it before killing another one)


This specific message `Processing 215 OneDrive items to ensure consistent state due to sync_list being used` was introduced as part of #659 then updated by #680 and #745 

It essentially means:
* Your using a sync_list file
* Because of Issue #658 and using sync_list - the 'delta' state of what OneDrive says has changed cannot be trusted, so a more indepth inspection is done. This is why you will see many 'larger' numbers with this message even when you may not have actually changed something.

Essentially because of the two items, the client is ensuring that you are in a consistent state - replicated locally what you are expecting, and if you move files via OneDrive HTTP interface into scope, these will now be downloaded and included correctly (this was the issue of #658) 

Long story short - nothing to worry about.

**Comment by SRGOM** [comment]
Thank you for the explanation! And the tldr of course :). I should have left a note rather than just emojis seeing as how incredibly helpful you've always been

**Comment by abraunegg** [comment]
@SRGOM
No problem - thanks for the feedback :)
